src/app.ts

```diff
@@ -85,11 +85,11 @@ export const initializeAppModules = () => {
 
 
     addModule({
-      name: "registerChangeSettingsHandler",
+      name: "registerEnterSettingsHandler",
```

Comment:
why this change.

Reply:
registerChangeSettingsHandler method has been renamed to registerEnterSettingsHandler in Teams JS SDK. On upgrading to 1.11.0-beta.2, the Yuri Test tab app build was failing as registerChangeSettingsHandler is no longer available in latest JS SDK. Its renamed to registerEnterSettingsHandler.